### PR TITLE
Hello World - Test repo

### PR DESCRIPTION
**Pull request by hajnaloltyan**
Created my first repo, using HTML and CSS, to request my first code review and open a pull request.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello World</title>
+  </head>
```
Comment (Felix45) [inline index.html:8]: - [ Required ] Please consider importing the `style.css` file into your `index.html` file as stated in the project requirements, at Microverse you'll work on a number of projects with different requirements, and for all these projects they'll have to satisfy the requirements. You can import this file to your `index.html` file as shown below.

```html
   <head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link href="style.css" rel="stylesheet" />
    <title>Hello World</title>
  </head>
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,63 @@
...
+- LinkedIn: [LinkedIn](https://linkedin.com/in/hajnaloltyan)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (Felix45) [inline README.md:61]: - [ Required ] I've noticed that the License link on your readme file does not lead to a License file in the project root directory, please consider adding a License file to your project named `LICENSE` and add the contents of the `MIT` License to the file, the contents of the MIT license can be found on this link [MIT](https://choosealicense.com/licenses/mit/). Finally, replace the License link on the readme file with the link shown below.

```
This project is [MIT](./LICENSE) licensed.
```

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @hajnaloltyan , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [x] Good descriptive commit messages :white_check_mark: 
- [x] Descriptive pull request title and summary :white_check_mark: 


## Required Changes ♻️
[Required] Please consider fixing the linter error on this pull request, a good pull request should always have all the linter checks passing with green checkmarks. Currently, one of your linter checks is failing.
Screenshot from 2023-03-22 00-11-53

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @hajnaloltyan    ,
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## To Highlight :100:
- Descriptive commit messages :heavy_check_mark:
- clean code :heavy_check_mark:

## Required Changes :recycle:

- [x] kindly consider removing the space in your readme file name so as it may appear on your branch cause currently its empty and this will make your repo look professional
not `README .md` its `README.md`

_Check the comments under the review._
## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!:clap::clap::clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @hajnaloltyan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Implemented all the changes from the previous reviewer :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**